analysis_templates/cms_minimal/law.cfg

```diff
@@ -52,11 +52,13 @@ chunked_io_pool_size: 2
 chunked_io_debug: False
 
 # csv list of task families that inherit from ChunkedReaderMixin and whose output arrays should be
-# checked for non-finite values before saving them to disk (right now, supported tasks are
-# cf.CalibrateEvents, cf.SelectEvents, cf.ProduceColumns, cf.PrepareMLEvents, cf.MLEvaluation,
-# cf.UniteColumns)
+# checked (raising an exception) for non-finite values before saving them to disk
```

Comment:
Since you changed the comment, does this mean that the finite value check is now available in all tasks?

Reply:
Yes, all tasks that produce columns have this check enabled now, so I wanted to remove the list in order for analyses, that started from the example at some point in the past, not to have an outdated list of tasks in their main cfg.